### DISCUSSION TITLE
一些界面上的功能建议

### DISCUSSION THREAD

**Discussion by falling7down** [xiaohucode/yidaRule#212] [Ideas]
1. 听书和详情页的目录加入拖动条，有些书籍章节太多，翻找太费劲
2. 目录中正在阅读的章节名用阴影显示不好看，建议改成字体加粗或变色
3. 视频界面的视频比例图标太像全屏显示了，容易误点，建议更换或者调整位置
4. 发现界面右上角的显示模式切换图标，和管理站点的分类图标太像了，很容易搞错位置，建议优化下图标或调换下位置

**Comment by 8721653** [comment]
大佬之前修的文件失效了请发我邮箱 3335774702@qq.com
